product_docs/docs/pgd/5/cli/command_ref/pgd_set-group-options.mdx

```diff
@@ -0,0 +1,66 @@
+---
+title: set-group-options
+---
+
+Sets group options such as enable_raft, enable_proxy_routing, and location.
+
+### Synopsis
+
+Following group options can be set through this command -
```

Comment:
Fixed but instead of back ticks used single quote. 

Unfortunately we cannot use back ticks in long description because it has special meaning in golang and we cannot escape back ticks inside back ticks. Following how we write it in golang. Multi line strings are already in back tick. 

(there is 1 way by concatenating strings but it has other problems like it introduces white spaces and also makes code ugly and unmanageable)

```
var setGroupOptionsCmd = &cobra.Command{
	Use:   "set-group-options",
	Short: "Sets group options such as 'enable_raft', 'enable_proxy_routing', and 'location'.",
	Long: `
You can set the following group options with this command:

- 'enable_raft'
- 'enable_proxy_routing'
- 'location'
- 'route_writer_max_lag'

Please note that 'enable_raft' and 'enable_proxy_routing' both should be true
if proxy is attached to the group.

Use 'pgd show-groups -o json' to view option values for each group.
`,
```